.npmignore

```diff
@@ -0,0 +1,9 @@
+test-results
```

Comment:
Ideally the `.npmignore` is a superset of the `.gitignore`, but there are some differences like `.idea`.

Reply:
I took the `.npmignore` from my other PR. `idea` was already in the `.gitignore`.